kubernetes/rbac.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   name: descheduler-cluster-role
   namespace: kube-system
 rules:
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["create"]
```

Comment:
You might want to extend the list with "update" as well. Event proceeding logic on the client side sometimes updates existing events instead of creating new one. Otherwise lgtm.

Reply:
@ingvagabund thanks for the review. I added "update" permissions in commit 6dd91b6a2289750b62bf535799887325ab22adb6.